app/views/trips/_people_to_allocate.html.erb

```diff
@@ -0,0 +1,6 @@
+<tr>
+  <td><%= trip.to_allocate(params[:id])["girls"]%></td>
+  <td><%= trip.to_allocate(params[:id])["boys"]%></td>
+  <td><%= trip.to_allocate(params[:id])["teachers"]%></td>
+  <td><%= trip.to_allocate(params[:id])["bus_drivers"]%></td>
+</tr>
```

Comment:
Same here

Reply:
This file is no currently used here. Also, it is a duplicate of another file